app/assets/stylesheets/components/home_page_header.scss

```diff
@@ -1,25 +1,24 @@
-header {
-    padding-bottom: 60px;
+.hero {
+  text-align: center;
+  width: 100%;
 
-  h1 {
-    margin-bottom: 30px;
+  header {
+    padding-bottom: 3.75rem;
```

Comment:
interesting... I guess i don't know that it REALLY matters, I just thought the main reason we used REMS was for flexibility, and that you wouldn't want your margins/padding to be flexible like that.

But I don't care either way.   I'm fine to leave it.